src/runtime_impl.cpp

```diff
@@ -415,6 +415,11 @@ namespace hpx
 
         lbt_ << "(1st stage) runtime_impl::start: booting locality " << here();
 
+	// Register this thread with the runtime system to allow calling
+	// certain HPX functionality from the main thread. Also calls
+	// registered startup callbacks.
+	init_tss("main-thread", 0, 0, "", "", false);
```

Comment:
Looks like the indentation got messed up here.

Reply:
Grr, good catch! Should be fixed now.